### PR TITLE
add env vars

### PR DESCRIPTION
**Pull request by hillalex**

### CODE REVIEW

File: config/orderly-web.yml [hunk 1]
```diff
@@ -54,6 +54,11 @@ orderly:
     repo: vimc
     name: orderly.server
     tag: master
+  env:
+    SLACK_URL: VAULT:slack/orderly-webhook:url
+    MONTAGU_USER: orderly
+    MONTAGU_HOST: db,
```
Comment (richfitz) [inline config/orderly-web.yml]: trailing comma here should be removed I think

File: config/science.yml [hunk 2]
```diff
@@ -9,6 +9,9 @@ orderly:
...
   ## from the vault, use the format VAULT:<path>:<key>
   env:
-    ORDERLY_DB_HOST: db_host
-    ORDERLY_DB_USER: db_user
-    ORDERLY_DB_PASS: VAULT:secret/db/password:value
+    MONTAGU_PASSWORD: VAULT:secret/database/science/users/orderly:password
+    ORDERLY_API_SERVER_IDENTITY: science
+    ANNEX_HOST: annex.montagu.dide.ic.ac.uk
+    ANNEX_PORT: 15432
+    ANNEX_PASSWORD: VAULT:secret/database/science/users/readonly:password
```
Comment (richfitz) [inline config/science.yml]: Should be `VAULT:secret/annex/users/readonly:password`

File: config/prod.yml [hunk 3]
```diff
@@ -9,6 +9,9 @@ orderly:
...
   ## from the vault, use the format VAULT:<path>:<key>
   env:
-    ORDERLY_DB_HOST: db_host
-    ORDERLY_DB_USER: db_user
-    ORDERLY_DB_PASS: VAULT:secret/db/password:value
+    MONTAGU_PASSWORD: VAULT:secret/database/production/users/orderly:password
+    ORDERLY_API_SERVER_IDENTITY: production
+    ANNEX_HOST: annex.montagu.dide.ic.ac.uk
+    ANNEX_PORT: 15432
+    ANNEX_PASSWORD: VAULT:secret/database/production/users/readonly:password
```
Comment (richfitz) [inline config/prod.yml]: Should be `VAULT:secret/annex/users/readonly:password`

**Review by richfitz — CHANGES_REQUESTED** [review]

**Review by EmmaLRussell — COMMENTED** [review]
I guess we don't need a value in github_org. 
Do we want to include 'master' in the tag values, or a placeholder that will be replaced by version tag by deployment script?

**Comment by hillalex** [comment]
> I guess we don't need a value in github_org.
> Do we want to include 'master' in the tag values, or a placeholder that will be replaced by version tag by deployment script?

I think this ticket was intended to work that out: https://vimc.myjetbrains.com/youtrack/issue/mrc-291

**Pull request merged**